src/index.js

```diff
@@ -118,14 +119,25 @@ class ReactTooltip extends Component {
     let targetArray = this.getTargetArray()
 
     let dataEvent
+    let dataEventOff
     for (let i = 0; i < targetArray.length; i++) {
       if (targetArray[i].getAttribute('currentItem') === null) {
         targetArray[i].setAttribute('currentItem', 'false')
       }
       dataEvent = this.state.event || targetArray[i].getAttribute('data-event')
       if (dataEvent) {
-        targetArray[i].removeEventListener(dataEvent, this.checkStatus)
-        targetArray[i].addEventListener(dataEvent, this.checkStatus, false)
+        dataEventOff = this.state.eventOff || targetArray[i].getAttribute('data-event-off');
+        // if off event is specified, we will show tip on data-event and hide it on data-event-off
+        if (dataEventOff) {
+          targetArray[i].removeEventListener(dataEvent, this.showTooltip);
```

Comment:
I don't think this can work because if you check the method `checkStatus`, you will find there are more things than showTooltip and hideTooltip. So it would be better if dataEventOff can work well with `checkStatus`(you don't have to use that method, I just mean maybe you can refer to it)